script.js

```diff
@@ -1,4 +1,218 @@
+// define options for game mode
+var gameStateArray = [
+  "RollDiceModePlayerOne",
+  "ChooseDiceOrderPlayerOne",
+  "RollDiceModePlayerTwo",
+  "ChooseDiceOrderPlayerTwo",
+];
+
+// set game mode to Player One's Dice Rolls
+var gameMode = gameStateArray[0];
+
+// define variables for each die roll and the players' final values after ordering
+// for player one
+var diceRollOnePlayerOne = "";
+var diceRollTwoPlayerOne = "";
+var playerOneNumber = "";
+// for player two
+var diceRollOnePlayerTwo = "";
+var diceRollTwoPlayerTwo = "";
+var playerTwoNumber = "";
+
+// leaderboard function
+var playerOneWins = 0;
+var playerTwoWins = 0;
+var leaderboardText = "";
+var leaderboard = function () {
```

Comment:
function naming could be clearer. maybe `printLeaderboard` or `generateLeaderboard`